week4/W4D4/live/mongoose/mongoose-intro.js

```diff
@@ -52,17 +57,18 @@ const pokemonSchema = new mongoose.Schema({
   moves: {
     type: [String],
     required: true,
+    set: stringArrayPaddingFactory(4),
     validate: {
       validator: (movesArray) => {
         return movesArray.length === 4;
       },
       message: "Pokemons should have 4 moves",
     },
-    default: ["", "", "", ""],
+    default: [],
   },
   pokeTypes: {
     type: [String],
-    set: completePokeTypes,
+    set: padToLength2,
```

Comment:
we don't we use that variable somewhere else please make it behave like you do for moves!